src/utils/colonyMotions.ts

```diff
@@ -13,17 +13,18 @@ export enum MotionVote {
 export const noMotionsVotingReputationVersion = 4;
 
 export enum MotionState {
-  Staked = 'Staked',
+  Supported = 'Supported',
   Staking = 'Staking',
   Voting = 'Voting',
   Reveal = 'Reveal',
-  Objection = 'Objection',
-  Motion = 'Motion',
+  Opposed = 'Opposed',
+  Motion = 'Motion', // @TODO: This was used by the old MotionTag component and should be removed
```

Comment:
Can we not remove this as part of this PR, we are in a strange state in the CDapp where we have left in obsolete parts that are then used by others for the v5, my preference would just be to remove this entirely to avoid this problem in the future